cluster/examples/kubernetes/ceph/crds.yaml

```diff
@@ -284,6 +284,8 @@ spec:
                         properties:
                           ip:
                             type: string
+                    externalMgrPrometheusPort:
+                      type: integer
```

Comment:
Should we add a minimum/maximum property?

Reply:
Like 1 to 65535

Reply:
yes, looks like we have that range on other ports in the schema as well